Makefile

```
@@ -195,7 +195,7 @@ issues: ## Run GitHub issue analysis scripts
 	./hack/github/label_issue_count.py > "karpenter-labels-$(shell date +"%Y-%m-%d").csv"
 
 website: ## Serve the docs website locally
-	cd website && npm install && git submodule update --init --recursive && hugo server
+	cd website && npm install && hugo server
```

```suggestion
	cd website && npm install && hugo server && hugo mod tidy
```

We should consider running `hugo mod tidy` to ensure that the `go.mod` is up-to-date here